pywemo/ouimeaux_device/dimmer.py

```diff
@@ -0,0 +1,32 @@
+from .switch import Switch
+
+class Dimmer(Switch):
+    def __init__(self, *args, **kwargs):
+        Switch.__init__(self, *args, **kwargs)
+        self._brightness = {}
+
+    def get_brightness(self):
+        """
+        Get brightness from device
+        """
+        brightness = self.basicevent.GetBinaryState().get('brightness')
+        self._brightness = brightness
+
```

Comment:
You can make this cache the value like `get_state` - which will stop it always calling the device unless you're explicit. 

Also I think you forgot to return the value.

Lastly - good to make sure the result is an int. I think wemo returns strings.

(I haven't tested this)
```
    def get_brightness(self, force_update):
        """
        Get brightness from device
        """
        if force_update or self._brightness is None:
            try:
                brightness = int(self.basicevent.GetBinaryState().get('brightness'))
            except ValueError:
                brightness = 0

        self._brightness = brightness
        return self._brightness
```

Reply:
should I have to put force_update=False in the first line? Thats the only way I can get that to work

Reply:
You're quite right - sorry.

Last thought is it will make the subscription logic in HA work  better if you add.

```
    def subscription_update(self, _type, _param):
        return False
```

This will tell HA that this object doesn't pull state from the update data.

Sorry for the slight random feeling comments - have been rushing around.

Reply:
I just made the changes in my branch, but will test here this afternoon. Thanks for all your help, definitely been a learning experience!